auto_configurator/conf/search_config/gpt3/175b.yaml

```diff
@@ -11,7 +11,7 @@ train_settings:
   tflops_per_gpu: 140  # Estimated tflops per GPU.
   num_tokens_in_b: 300  # Unit in billions, typically 300B for GPT3 models.
   vocab_size: 51200
-  seq_length: 2048 # available seq_length list for GPT-3 models: [2048, 4096, 8192]
+  seq_length: 2048 # available seq_length list for GPT-3 models: [2048, 4096, 8192, 16384, 32768]
```

Comment:
I thought these large seq_lens were only supported for models of <=20.6B params. If that's the case please update this comment for 175b and the smaller model sizes too.